### PR TITLE
[Transforms][NFC] Tiny fixes in SplitModule

### PR DESCRIPTION
**Pull request by sergachev**
Fix repeated map lookup, variable name, formatting and a missing space.

### CODE REVIEW

File: llvm/lib/Transforms/Utils/SplitModule.cpp [hunk 1]
```diff
@@ -275,8 +278,8 @@ void llvm::SplitModule(
     ValueToValueMapTy VMap;
     std::unique_ptr<Module> MPart(
         CloneModule(M, VMap, [&](const GlobalValue *GV) {
-          if (ClusterIDMap.count(GV))
-            return (ClusterIDMap[GV] == I);
+          if (auto It = ClusterIDMap.find(GV); It != ClusterIDMap.end())
+            return (It->second == I);
```
Comment (fhahn) [inline llvm/lib/Transforms/Utils/SplitModule.cpp:282]: nit: no () needed

File: llvm/lib/Transforms/Utils/SplitModule.cpp [hunk 2]
```diff
@@ -275,8 +278,8 @@ void llvm::SplitModule(
     ValueToValueMapTy VMap;
     std::unique_ptr<Module> MPart(
         CloneModule(M, VMap, [&](const GlobalValue *GV) {
-          if (ClusterIDMap.count(GV))
-            return (ClusterIDMap[GV] == I);
+          if (auto It = ClusterIDMap.find(GV); It != ClusterIDMap.end())
```
Comment (fhahn) [inline llvm/lib/Transforms/Utils/SplitModule.cpp:281]: For readability it would probably be better to move out the assignment of It.

Comment (sergachev) [reply llvm/lib/Transforms/Utils/SplitModule.cpp:281]: Would it? It's a common pattern that reduces the scope of 'It': https://github.com/llvm/llvm-project/blob/9d45077df92a64ff8db5b249ac2f2d45e82ab73c/clang/lib/AST/Interp/Program.cpp#L117, https://github.com/llvm/llvm-project/blob/9d45077df92a64ff8db5b249ac2f2d45e82ab73c/llvm/lib/IR/DebugLoc.cpp#L79, https://github.com/llvm/llvm-project/blob/9d45077df92a64ff8db5b249ac2f2d45e82ab73c/clang/lib/Analysis/FlowSensitive/Formula.cpp#L60 etc.

File: llvm/lib/Transforms/Utils/SplitModule.cpp (hunk 1 above)
Comment (sergachev) [reply llvm/lib/Transforms/Utils/SplitModule.cpp]: Done

**Comment by sergachev** [comment]
ping

File: llvm/lib/Transforms/Utils/SplitModule.cpp (hunk 2 above)
Comment (Pierre-vh) [reply llvm/lib/Transforms/Utils/SplitModule.cpp:281]: I also use this pattern a lot and I think it's more readable than 2 separate line

**Review by Pierre-vh — APPROVED** [review]

File: llvm/lib/Transforms/Utils/SplitModule.cpp (hunk 2 above)
Comment (fhahn) [reply llvm/lib/Transforms/Utils/SplitModule.cpp:281]: Right I guess sits down to personal preference.

**Review by fhahn — APPROVED** [review]
LGTM, thanks!

**Pull request merged**